compute/elastic-metal/concepts.mdx

```diff
@@ -78,9 +78,17 @@ A Region is represented as a Geographical area such as France (Paris: `fr-par`)
 
 <Concept>
 
+## Remote access
+
+Remote access allows you to access the keyboard, video, and mouse of the machine remotely by using [KVM over IP device](https://en.wikipedia.org/wiki/KVM_switch#KVM_over_IP_(IPKVM)). It enables you access to the machine even in case the installed OS is not working properly, to debug the system or to install an own OS from a remote ISO file. Remote access is available as option on most Elastic Metal offers.
```

Comment:
```suggestion
Remote access allows you to access the keyboard, video, and mouse of the machine remotely by using [KVM over IP device](https://en.wikipedia.org/wiki/KVM_switch#KVM_over_IP_(IPKVM)). It enables you access to the machine even in case the installed OS is not working properly, to debug the system, or to install your own OS from a remote ISO file. Remote access is available as an option on most Elastic Metal offers.
```